backend/tests/integration/records/serializers/json/test_literature.py

```diff
@@ -900,3 +900,31 @@ def test_record_link_to_hepdata_in_detail_view(inspire_app):
         response.json["metadata"]["external_system_identifiers"]
         == expected_response_metadata
     )
+
+
+def test_literature_detail_links(inspire_app):
+    expected_status_code = 200
+    record = create_record("lit")
+    cn = record["control_number"]
+    expected_links = {
+        "bibtex": f"http://localhost:5000/literature/{cn}?format=bibtex",
+        "citations": f"http://localhost:5000/literature/{cn}/citations",
+        "json": f"http://localhost:5000/literature/{cn}?format=json",
+        "latex_eu": f"http://localhost:5000/literature/{cn}?format=latex_eu",
+        "latex_us": f"http://localhost:5000/literature/{cn}?format=latex_us",
+    }
```

Comment:
could you add tests for `/api/arxiv` and `/api/orcid`?

Reply:
:+1: